ctrd/container.go

```diff
@@ -357,13 +357,13 @@ func (c *Client) createContainer(ctx context.Context, ref, id string, container
 	logrus.Infof("success to get image %s, container id %s", img.Name(), id)
 
 	// create container
+	//
+	// TODO: The github.com/containerd/containerd/oci provides oci.WithMounts
+	// in v1.1.0. Before we upgrade it, we use assign way to handle the case.
+	container.Spec.Mounts = newOCILinuxMounts()
```

Comment:
@fuweid , can you put these logic into file like `daemon/mgr/spec_mount.go`, since this is oci spec related ?